packages/token-list/src/index.js

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+const { toChecksumAddress } = require('web3-utils')
+const memoize = require('lodash/memoize')
+const sushiswapTokens = require('@sushiswap/default-token-list').tokens
+const uniswapTokens = require('@uniswap/default-token-list').tokens
+const vesperTokens = require('vesper-metadata/src/vesper.tokenlist.json').tokens
+
+const zapperTokens = require('./token-lists/zapper.json').tokens
+// Source at https://github.com/pangolindex/tokenlists
+const pangolindexTokens = require('./token-lists/pangolindex.json').tokens
+const customTokens = require('./token-lists/custom.json').tokens
+const hemiTokens = require('./token-lists/hemi.json').tokens
+
+const getTokenListByChain = memoize(chainId =>
+  []
+    .concat(customTokens)
+    .concat(vesperTokens)
+    .concat(hemiTokens)
+    .concat(uniswapTokens)
+    .concat(sushiswapTokens)
+    .concat(zapperTokens)
+    .concat(pangolindexTokens)
+    .filter(t => t.chainId === chainId)
+    .map(t => ({ ...t, address: toChecksumAddress(t.address) }))
+)
+
+const findTokenByAddress = memoize(
+  function (_address, chainId = 1) {
+    const tokenlist = getTokenListByChain(chainId)
+    return tokenlist.find(
+      ({ address }) => address.toLowerCase() === _address.toLowerCase()
+    )
+  },
+  (...args) => JSON.stringify(args)
+)
+
+const findTokenBySymbol = memoize(
+  function (_symbol, chainId = 1) {
```

Comment:
Should we default here to Hemi mainnet? (Found the id! its 43111)
```suggestion
  function (_symbol, chainId = 43111) {
```

or even Hemi Sepolia?

```suggestion
  function (_symbol, chainId = 743_111) {
```

Reply:
Yes, we should.

Reply:
changed here: https://github.com/hemilabs/pure.finance/pull/2/commits/9aefa58924e45d52cc6e24097b0b9dde3e03e27a